catboost/python-package/ut/medium/test.py

```diff
@@ -7108,3 +7109,24 @@ def test_to_regressor_wrong_type():
     model.fit(train_pool)
     with pytest.raises(CatBoostError):
         to_regressor(model)
+
+
+def test_load_and_save_quantization_borders():
+    borders1_file = test_output_path('borders1.dat')
+    borders2_file = test_output_path('borders2.dat')
+    borders3_file = test_output_path('borders3.dat')
```

Comment:
Let's give them more meaningful names like 'borders_32.dat', 'borders_10.dat', 'borders_with10.dat'.
And for 'pool1', 'pool2', 'pool3' as well.